CHANGELOG.md

```diff
@@ -30,6 +30,9 @@ __Special thanks to Albert Teoh and Sameera Perera for contributing to this rele
 - Fixes a bug in Simplify where invalid interior rings would be omitted rather
   than producing an error.
 
+- Adds a wrapper in the `geos` package for the `GEOSMakeValid_r` function
```

Comment:
What was the motivation for wrapping the `GEOSMakeValid_r` function? What challenges are there to implementing this natively in Go?

Reply:
It's something that Ken _may_ find useful in his current work (might be better to take that discussion offline for further details).

The main challenges involved there would be coming up with an actual algorithm to do it (possibly that's the hard part) and the implementing it. It would most likely involve a [DCEL](https://en.wikipedia.org/wiki/Doubly_connected_edge_list) data structure (which we have already implemented). I'd ballpark estimate 50hrs to 100hrs of effort would be required.

Reply:
If there are plans to do this, maybe it's worth creating a ticket for this work; but if it's not in scope of the goals of simplefeatures, then all good.

Reply:
Good call, it's definitely in scope. I've created a ticket here: https://github.com/peterstace/simplefeatures/issues/435